app/controllers/feed_controller.rb

```diff
@@ -0,0 +1,14 @@
+class FeedController < ApplicationController
+  skip_before_action :authenticate_user!
+
+  # GET /feed.rss
+  # GET /feed.atom
+  def index
+    @talks = Talk.all.order(created_at: :desc).includes(:speakers).limit(100)
```

Comment:
We could add here in memory caching or persist on disk based on some kind of checksum. Though as SQLite is used in production, not sure if there is going to be a big impact of that. Let me know what you think about this.

Reply:
Rails cache is available with this app using Litestack cache (using Sqlite behind the scene). 
You could cache the views using normal Rails cache tools but not sure it really worth it at this point

Reply:
Yeah, makes sense. Then lets skip premature optimisation if its not needed.